### ISSUE TITLE
Links that open in a new window should be annotated in the DOM

### ISSUE THREAD

**Issue by fg-j** [paketo-buildpacks/paketo-website#196]
Links that open in new windows can create a disorienting experience for users with low vision, those who are using a screen reader or other device that expresses page content in a non-visual way, and those who prefer reduced motion. It is a best practice to [inform users that links open in a new window](https://codersblock.com/blog/external-links-new-tabs-and-accessibility/#how-to-inform-your-users).

## A/C
`AS` A user who uses a screen-reader or similar technology to interpret page content
`WHEN` I focus a link anywhere on the site that opens in a new window
`THEN` I'm aware of this before I activate the link because there is a `<span>` inside the link that alerts me

AND

`AS` A user interacting with the Paketo site using a mouse
`WHEN` I see a link that opens in a new window
`THEN` There is nothing visually different about the link as compared to other ones.

Note: It is in fact an accessibility best practice to visually signal that links open in a new window as well (for instance, including an SVG icon that represents this). We should discuss the second A/C and consider including some visual marker for these sorts of links.
